### PR TITLE
[FP-11]Private routing

### PR DESCRIPTION
**Pull request by thisisofficial**
### Description

Added a component for easy implementaition of new private routes with simple logic.
Also prepped some file for betteruse of this feature, this changes went unused due to unsufficient functionality on the Login part.

This, however, will be updated in due time once the Login page gets full functionality, or once I figure a way to use redux to do it for me. Whichever may come first.

### Testing Checklist

- [ ] Test cases have been added/updated
- [x] Tested locally in browser (at least one other than Chrome)
- [x] All tests pass

### What JIRA Ticket(s) does this PR address?

- [FP-11](https://cd95.atlassian.net/browse/FP-11)

### CODE REVIEW

File: src/routes/routes.tsx [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
 export function Router() {
+  const [userLog, setUserLog] = useState(false);
   return (
-    <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<DataGrid />} />
-        <Route path="/login" element={<Login />} />
-      </Routes>
-    </BrowserRouter>
+    <loginContext.Provider value={{ userLog, setUserLog }}>
```
Comment (caarlosdamian) [inline src/routes/routes.tsx]: Eliminar Context ya que se usara Redux

File: src/hooks/loginContext.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const loginContext = React.createContext({});
```
Comment (caarlosdamian) [inline src/hooks/loginContext.tsx]: no necesitamos context

**Review by caarlosdamian — CHANGES_REQUESTED** [review]

File: src/hooks/loginContext.tsx (hunk 2 above)
Comment (thisisofficial) [reply src/hooks/loginContext.tsx]: Deleted.

File: src/routes/routes.tsx (hunk 1 above)
Comment (thisisofficial) [reply src/routes/routes.tsx]: Removed.

**Review by caarlosdamian — APPROVED** [review]

**Pull request merged**